lib/project.js

```diff
@@ -113,14 +113,17 @@ function Project$get(key, dflt) {
             _.get(this.rawPkgJSON, key, dflt));
 }
 
+var GH_SSH = /^(?:git\+ssh:\/\/)?git@github.com[:\/]([^/]+)\/([^/]+?)(\.git)?$/;
```

Comment:
Any reason not to use URL.parse?

Also, having unit tests specifically for these regexes might be useful.